portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,72 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Nnenna's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+ <body>
+
+     
+    <h1>Nnenna Dara</h1>
+
+    <div class="tab">
+        <button class="tablinks" onclick="openCity(event, 'About Me')" id="defaultOpen">About Me</button>
+        <button class="tablinks" onclick="openCity(event, 'Job Experience')">Job Experience</button>
+        <button class="tablinks" onclick="openCity(event, 'Extracurriculars')">Extracurriculars</button>
+        <button class="tablinks" onclick="openCity(event, 'Projects')">Projects</button>
     </div>
-  </body>
+ 
+    <div id="About Me" class="tabcontent">
+            <h1>About Me</h1>
+            <p style="color:DodgerBlue;">This is my portfolio page.</p>
+            <hr>
+            <img src="/images/picture1.jpg" alt="A picture of me!" style="width:500px;height:550px;"> >
+            <p>I am a rising junior at Cornell University trying to
+                pursue an independent major, combining the curriculum of 
+                computer science and mechanical engineering.I am interning
+                at Google as a STEP intern this summer. I am interested in 
+                software development, robotics and product management.</p>
+            <hr>
+            <p>My <a href="https://www.linkedin.com/in/nnenna-dara-227328179/">LinkedIn</a></p>
+            <p>My <a href="https://github.com/nnenna352">GitHub</a></p>
+
+            <p>Click here to get a random fun fact about me:</p>
+            <button onclick="addRandomGreeting()">Click here</button>
```

Comment:
Since its not a greeting, rename this along those lines to addRandomFunFact or addRandomFact maybe. Also change wherever it says 'greeting' in the JS methods to similar naming.